### PR TITLE
Add missing portuguese translation

### PR DESCRIPTION
**Pull request by ppelegrin**
### Issues

- None

### Todo

- [X] None

### CODE REVIEW

**Comment by ogerly** [comment]
nice

File: webapp/locales/pt.json [hunk 1]
```diff
@@ -41,193 +136,733 @@
...
+          "no-email-request": "Are you certain that you requested a change of your email address?"
+        },
+        "support": "If the problem persists, please contact us by email at"
+      }
+    },
+    "validation": {
+      "slug": {
+        "regex": "Allowed characters are only lowercase letters, numbers, underscores and hyphens.",
+        "alreadyTaken": "This user name is already taken."
+      }
```
Comment (roschaefer) [inline webapp/locales/pt.json]: This seems to be English, not Portuguese. :thinking:

File: webapp/locales/pt.json [hunk 2]
```diff
@@ -41,193 +136,733 @@
...
+      "title": "By humans for humans",
+      "description": "Please help us to get new donators for Human Connection, so the network can take off as soon as possible. <a href=\"https://human-connection.org/\" target=\"_blank\"> https://human-connection.org </a>"
+    },
+    "have-fun": "Now have fun with the alpha version of Human Connection! For the first universal peace. ♥︎",
+    "closing": "Thank you very much <br> <br> your Human Connection Team"
   }
 }
+
+// {
+//   "password-reset": {
```
Comment (roschaefer) [inline webapp/locales/pt.json]: There is no comment in JSON. This will certainly be invalid syntax.

File: webapp/locales/pt.json (hunk 1 above)
Comment (ppelegrin) [reply webapp/locales/pt.json]: I'm still working on it ;)

File: webapp/locales/pt.json (hunk 2 above)
Comment (ppelegrin) [reply webapp/locales/pt.json]: I just copy and commented these out to use existing translations

**Pull request merged**